apisix/plugins/real-ip.lua

```diff
@@ -84,6 +103,17 @@ local function get_addr(conf, ctx)
             return addrs
         end
 
+        if conf.recursive and conf.trusted_addresses then
+            local split_addrs, _ = ngx_re_split(addrs, ",\\s*", "jo")
```

Comment:
`local split_addrs = ngx_re_split(addrs, ",\\s*", "jo")` better? Or deal with the error?

Reply:
Thanks for the review, the above if condition has already dealt with the absence of commas, so there should be no need to deal with err here? I'm not very familiar with lua, but this `_` may indeed be removed.